src/dso_api/settings.py

```diff
@@ -405,3 +405,9 @@
 HAAL_CENTRAAL_API_KEY = os.getenv("HAAL_CENTRAAL_API_KEY", "UNKNOWN")
 HAAL_CENTRAAL_KEYFILE = os.getenv("HC_KEYFILE")
 HAAL_CENTRAAL_CERTFILE = os.getenv("HC_CERTFILE")
+
+SHELL_PLUS_POST_IMPORTS = [
+    "from django.apps.registry import apps",
+    "from dso_api.dynamic_api.filterset import filterset_factory",
+    "from pprint import pprint",
+]
```

Comment:
This could be regarded as nitpicking, but I am a firm believer in that code should express intention. Here I see a constant (because the variable is in all caps), but with a mutable data structure (a list) assigned to it. Using a mutable data structure instead of an immutable one suggests, at least to me, that mutability is required. But that contradicts its assignment to a constant.

So I would either suggest, not using a constant here (`SHELL_PLUS_POST_IMPORTS` -> `shell_plus_post_imports`) or use an immutable datastructure (eg a tuple) instead of a list.

Reply:
The same reasoning can be applied to https://github.com/Amsterdam/dso-api/pull/392/files#diff-6b63d8df4a47199509b132ee48d6a28acb9e7daf592fe5bfbb4c20dcd03d79d9R116.

```python
if lookup not in ["contains", "exact"]:
    ...
```
Testing for membership does not require a mutable datastructure. I would write that as:

```python
if lookup not in ("contains", "exact"):
    ...
```
But this is even more nitpicking than then the constant versus list assigment I made above.

Reply:
done :)